schema/data/project-page-data.jsonld

```diff
@@ -0,0 +1,242 @@
+{
+  "id": "http://dev.app.regen.com/projects/38",
+  "environmentalAuthority": {
+    "name": "Corantioquia",
+    "url": "http://corantioquia.gov.co/"
+  },
+  "marketType": "rfs:ComplianceMarket",
+  "bioregion": [
+    "Amazon Basin"
+  ],
+  "biomeType": [
+    "TropicalForest"
+  ],
+  "watershed": [
+    "Amazon River"
+  ],
+  "subWatershed": [
+    "Upper Amazon"
+  ],
+  "ecosystemType": [
+    "rfs:CloudForest"
+  ],
+  "offchainCreditsInfo": {
+    "creditsRegistered": {
+      "numericValue": 1000.0,
+      "unit": "unit:HA"
+    },
+    "creditsAvailable": {
+      "numericValue": 800.0,
+      "unit": "unit:HA"
+    },
+    "creditsRetired": {
+      "numericValue": 200.0,
+      "unit": "unit:HA"
+    }
+  },
+  "projectDuration": "P1Y",
+  "projectDurationMinimum": "P6M",
+  "projectDurationMaximum": "P2Y",
```

Comment:
I'm not sure that we need minimum and maximum defined at the project level.  I believe this is diff btw protocol v3 and v4, and if that's the only thing that would change min & max duration, then maybe we can infer this from the protocol, and explicitly set the protocol version somewhere in the project page metadata?

Reply:
projectDurationMaximum/Minimum will be stored at the credit class metadata level so let's just add creditClassVersion in the project metadata

Reply:
In v4 of the protocol, @clevinson @blushi @paul121, the min duration is 20 years, the max is 50. Technically they still could re-register so the max can be extended. I'm wondering though, if it makes more sense to put the duration min and max at the protocol level and place the project term here as they should specify a term in the project plan...

Reply:
> projectDurationMaximum/Minimum will be stored at the credit class metadata level 

This meant to put duration min/max at the protocol level indeed (but we just preferred to use the technical terms from Regen Ledger to be clear, ie credit class), while the project duration/term would be in the project metadata.
Is this what you meant? @S4mmyb